src/sparseml/keras/utils/pytorch2keras.py

```diff
@@ -0,0 +1,347 @@
+import re
```

Comment:
let's move this script over to the utils folder under root since it's a utility script and not meant to be imported anywhere